script.js

```diff
@@ -0,0 +1,82 @@
+
+var x = "x";
+var o = "o";
+var counter = 0;
+var allSquares = document.getElementsByClassName('div.gridSquare');
+
+for(counter = 0; counter < allSquares; i++){
+  allSquares[counter].addEventListener("click", changeSquare);
```

Comment:
Great job with this approach! Like that you didn't repeat the code to add an event listener 9 times.